mappings/net/minecraft/entity/vehicle/ExperimentalMinecartController.mapping

```diff
@@ -0,0 +1,65 @@
+CLASS net/minecraft/unmapped/C_btculrky net/minecraft/entity/vehicle/ExperimentalMinecartController
+	FIELD f_hikwcdre lastLerpStep Lnet/minecraft/unmapped/C_btculrky$C_xoihtwzz;
+	FIELD f_icggxsas Y_OFFSET_ON_RAIL D
+	FIELD f_jbgmphca currentLerpSteps Ljava/util/List;
+	FIELD f_kxhjtdkb REFRESH_TICKS I
+	FIELD f_ljrpmnlw currentTotalWeight D
+	FIELD f_mnqowxmz lerpSteps Ljava/util/List;
+	FIELD f_rhqsxplj ticksToNextUpdate I
+	FIELD f_tbzdvaef cachedTicksToNextUpdate I
+	FIELD f_uqdjwxfd cachedPartialTick F
+	FIELD f_yhexcgfs lastPartialTickStep Lnet/minecraft/unmapped/C_btculrky$C_dzjsmtju;
+	FIELD f_zosxuhfo V_SHAPED_TRACK_STOP_THRESHOLD D
+	METHOD m_djwrnguw getLerpedPitch (F)F
+	METHOD m_hptwubgx boostFromPoweredRail (Lnet/minecraft/unmapped/C_vgpupfxx;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_txtbiemp;)Lnet/minecraft/unmapped/C_vgpupfxx;
+		ARG 1 velocity
+	METHOD m_hzzcyntk decelerateFromPoweredRail (Lnet/minecraft/unmapped/C_vgpupfxx;Lnet/minecraft/unmapped/C_txtbiemp;)Lnet/minecraft/unmapped/C_vgpupfxx;
+		ARG 1 velocity
+	METHOD m_izllixuq setRotation (FF)V
+		ARG 1 yaw
+		ARG 2 pitch
+	METHOD m_jbchdtyf setLastLerpStep ()V
+	METHOD m_kmamcsdp restAtVShape (Lnet/minecraft/unmapped/C_wqmdpcsg;Lnet/minecraft/unmapped/C_wqmdpcsg;)Z
```

Comment:
```suggestion
	METHOD m_kmamcsdp restIfInValley (Lnet/minecraft/unmapped/C_wqmdpcsg;Lnet/minecraft/unmapped/C_wqmdpcsg;)Z
```
maybe?
imo the idea is more obvious, but maybe people would think it includes things like `\__/`

I think it should include "if" or "try" though

Reply:
i think valley makes it a little too generic, yeah
i'll go with `restIfInVShape`